### ISSUE TITLE
Use exact value of `hiddenInput` option

### ISSUE THREAD

**Issue by ameotoko** [jackocnr/intl-tel-input#1492]
Is there a way to give a hidden input a name with exact value of `hiddenInput`? i.e. not to force-wrap it in square brackets, if original input name is bracketed.

**Comment by jackocnr** [comment]
Not currently, but there probably should be! I think ideally the way `hiddenInput` should work is to accept a function, which is passed the input name as an argument, in case devs want to reference that e.g. 

```js
hiddenInput: function(inputName) {
    return `${inputName}[full]`;   // or in your case just return a string here e.g. "phone_number"
}
```

I don't have time to implement this myself right now, but I would be open to a pull request that adds this functionality.

**Comment by loverg-c** [comment]
if you don't care about keeping a name on the iti input: 

```
const originalName = input.name;
input.name = ''

let iti = intlTelInput(input, {
            utilsScript: utils,
            hiddenInput: originalName,
        });
```

**Comment by jackocnr** [comment]
Implemented in v19.0.0
